node_modules/oae-admin/manageuser/manageuser.html

```diff
@@ -98,6 +83,43 @@ <h3 id="manageuser-editprofile-header-container"><!-- --></h3>
     </div>
 </div>
 
+<div id="manageuser-authentication-template"><!--
+    {if loginIds}
```

Comment:
Do we need to check this? I feel like we can always have a `loginIds` property available.